### PR TITLE
LAW-310: Add FB script to each embed iframe

### PR DESCRIPTION
**Pull request by dashaluna**

### CODE REVIEW

**Comment by dashaluna** [comment]
@mattheu the script is now inserted into each `iframe` for FB embed. However, they don't seem to work. Not sure what I'm missing. Could you please have a look?

File: simple-facebook-instant-articles.php [hunk 1]
```diff
@@ -405,10 +406,65 @@ public function api_galleries_shortcode( $atts ) {
...
+						break 2;
+					}
+
+					unset( $facebook_handlers_ids[ $name ] );
+				}
+			}
+		}
+
+		// It is a facebook embed - insert script.
+		if ( $is_facebook_embed && function_exists( 'jetpack_facebook_add_script' ) ) {
```
Comment (mattheu) [inline simple-facebook-instant-articles.php]: might be best to have the function exists check right at the top of the function? No point in doing the whole find the embed handler dance if the function doesn't exist.

**Comment by mattheu** [comment]
OK I looked into this a little bit
1. Confirmed it doesn't work. FB complains about the `<post>` tag jetpack is outputting.
2. WP.com embeds DO NOT use the same markup. Adding our script hack to these embeds works great!

Not sure what to do here. We don't know what the embed logic is on .com. We could open a ticket on zendesk and ask for help?

**Comment by dashaluna** [comment]
Let's ask - it would be good to know what's going on!

File: simple-facebook-instant-articles.php (hunk 1 above)
Comment (dashaluna) [inline simple-facebook-instant-articles.php]: Good point I'll move :+1:

**Comment by dashaluna** [comment]
Closing in favour of https://github.com/humanmade/Simple-Instant-Articles-for-Facebook/pull/52

**Pull request closed without merging**